src/ivan_cherednikov_ex/ex02/bubble_sort.py

```diff
@@ -0,0 +1,18 @@
+def bubble_sort(input_data):
+    nums = len(input_data)
+    list_sorted = list(input_data)
+    for x in range(nums):
+        for z in range(0, nums - x - 1):
```

Comment:
If range starts with 0, you don't need to explicitly tell the function about that.